src/keyValue_test.cc

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2021, USC
+// All rights reserved.
+
+#include "keyValue.h"
+
+#include <gtest/gtest.h>
+
+namespace backend {
+// test initialization of backend
+TEST(CreateKeyValue, Initialization) {
+  KeyValue backend;
+  backend.put('a', 'b');
```

Comment:
This doesn't appear to actually test what was put into the store. Add an EXPECT or ASSERT.

Reply:
Yeah this was a bad test.  I will add more sensible testing in the next PR